cloud/passportauth/example/UserRepository.ts

```diff
@@ -22,23 +22,20 @@ const userSeedData = [
 ];
 
 /**
- * A sample user implementation
+ * A sample user service implementation
  */
-export class BaseUser implements User {
-  // Wrap user object
-  constructor(readonly user: any) {
+export class SampleUserService implements UserService {
+  // Map user object
+  public getLoginId(user: any) {
+    return user.username;
   }
 
-  public getLoginId() {
-    return this.user ? this.user.username : undefined;
+  public getPassword(user: any) {
```

Comment:
[Optional] If that's service we may even think about having password matcher method here as this will be always passed by customers. Getting password method may return password hash.